lib/mini_defender/rules/integer.rb

```diff
@@ -2,15 +2,16 @@
 
 class MiniDefender::Rules::Integer < MiniDefender::Rule
   def self.signature
-    'integer'
+    "integer"
   end
 
   def coerce(value)
     value.to_i
   end
 
   def passes?(attribute, value, validator)
```

Comment:
Thanks for the feedback! While using Integer(value) directly seems cleaner, it might not cover cases where the value is a float. In such cases, Integer(value) would truncate the decimal part instead of raising an error. Adding the check ensures consistent validation across all input types.

Reply:
Hala @Salman778, change it to `Integer(value.to_s)` this will ensure to fail when float is encountered